From 5164e735c32d227050f753e197c5adcf54ba4dfe Mon Sep 17 00:00:00 2001
From: Vladyslav Movchan <vladislav.movchan@gmail.com>
Date: Tue, 8 Apr 2025 11:39:19 +0300
Subject: [PATCH] Avoid duplication of `index.md` in `SUMMARY.md`

This fixes the following `mdbook build` error when using mdbook 0.4.48:
Error: Summary parsing failed for file="/tmp/wezterm/docs/./SUMMARY.md"
Caused By: Duplicate file in SUMMARY.md: "index.md"
---
 ci/generate-docs.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git ci/generate-docs.py ci/generate-docs.py
index f32b87d2ed7..6e73d8fbc23 100644
--- ci/generate-docs.py
+++ ci/generate-docs.py
@@ -508,6 +508,6 @@ def render(self, output, depth=0, mode="mdbook"):
 
 
 with open("SUMMARY.md", "w") as f:
-    f.write("[root](index.md)\n")
+    f.write("[root]\n")
     for page in TOC:
         page.render(f, depth=1, mode="mdbook")
